prow/jobs/eventing-webhook/eventing-webhook-certificates-build.yaml

```diff
@@ -0,0 +1,95 @@
+# Code generated by rendertemplates. DO NOT EDIT.
+
+
+presubmits: # runs on PRs
+  kyma-project/api-gateway:
+    - name: pull-eventing-webhook-certificates-build
+      annotations:
+        pipeline.trigger: "pr-submit"
```

Comment:
Please add the following annotations:
- `owner` with a name of a group or person responsible for this job
- `description` with a brief description of purpose of this job

Reply:
added the annotations to all relevant jobs